[WIN] Correction of build issues.

malavv opened the pull request:
Trying to build Brewtarget on Windows revealed a few issues. I will try to find and eliminate them. Here are these included in this pull request.
* Inverted logic on Multimedia DLL inclusion.
* MINGW_BIN having a redundant bin path.

rocketman768 commented on CMakeLists.txt:
```diff
@@ -404,7 +404,7 @@ IF( WIN32 )
+   IF(${NO_QTMULTIMEDIA})
       SET( Qt_DLLs,
         ${Qt_DLLs}
         ${QtMultimedia_location}
```
This is adding QtMultimedia DLLs. I think we should be adding these DLLs unless NO_QTMULTIMEDIA is ON, right?

malavv replied:
Thanks Phillip. You're completely right. I got bitten by the double negative. There is still an error because the original version does not seem to work but I'll revert that change.

malavv commented:
Hi Phillip and thanks again for looking at this. Here are the new modifications : 
* I reverted the double negative.
* I found the error which I was seeing (a trailing comma)
* I modified a comment about the bin folder to indicate the inclusion of the "bin".

malavv commented:
Ready for review and comments.

rocketman768 commented:
:+1:

Go ahead and squash the commits together.
